releasenotes/notes/add-maas-checks-for-filbeat-es-rsyslog-0289a1d9a1c198d0.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    This adds MaaS checks for elasticsearch, beaver, and 
```

Comment:
Thank you for updating the release note but it's my understanding that liberty and earlier don't use release notes. Is that correct @git-harry ?